Dockerfile

```diff
@@ -1,19 +1,43 @@
-# reference https://developers.google.com/web/tools/puppeteer/troubleshooting#setting_up_chrome_linux_sandbox
-FROM node:current-alpine
+FROM node:14-slim
 
-# manually installing chrome
-RUN apk add chromium
+# Install latest chrome dev package and fonts to support major charsets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+# installs, work.
+RUN apt-get update \
+    && apt-get install -y wget gnupg \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y libxshmfence1 libglu1 google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 \
+    --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
 
-# skips puppeteer installing chrome and points to correct binary
-ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true \
-    PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser
+# Install puppeteer so it's available in the container.
+RUN npm init -y &&  \
+    npm i puppeteer \
+    # Add user so we don't need --no-sandbox.
+    # same layer as npm install to keep re-chowned files from using up several hundred MBs more space
+    && groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
+    && mkdir -p /home/pptruser/Downloads \
+    && mkdir -p /home/pptruser/app \
+    && mkdir -p /home/pptruser/config \
+    && chown -R pptruser:pptruser /home/pptruser \
+    && chown -R pptruser:pptruser /home/pptruser/app \
+    && chown -R pptruser:pptruser /home/pptruser/config \
+    && chown -R pptruser:pptruser /node_modules \
+    && chown -R pptruser:pptruser /package.json \
+    && chown -R pptruser:pptruser /package-lock.json
 
-WORKDIR /app
+# Run everything after as non-privileged user.
+USER pptruser
+
+WORKDIR /home/pptruser/app
 COPY ["package.json", "package-lock.json*", "./"]
+
 RUN npm ci
 # the rest of your dockerfile here
 COPY . .
 
-ENV IBFI_CONFIG_PATH='/config'
+ENV IBFI_CONFIG_PATH='/home/pptruser/config'
```

Comment:
this env var is not longer in use, 
It should be the yaml config file and should be `CONFIG_FILE` (the yaml file and not only directory)

Reply:
In your case, you set CONFIG_FILE in docker-compose, so we can just remove this like. It will be great if you do that.